src/azul/version_service.py

```diff
@@ -0,0 +1,74 @@
+from typing import Optional
+
+import boto3
+
+from azul import config
+
+
+class VersionService:
+    key_name = 'object_url'
+    value_name = 'current_version'
+
+    def __init__(self):
+        self.client = boto3.client('dynamodb', 'us-east-1')
```

Comment:
```suggestion
        self.client = boto3.client('dynamodb')
```

Generally speaking, we cannot hard-code the region. The region is configured like so

https://github.com/DataBiosphere/azul/blob/develop/deployments/.example.local/environment#L38

Reply:
Is there a more elegant way to read the value from within the codebase or is it okay to just use `os.environ` here?

Reply:
I'm pretty sure, you don't have to specify anything. You won't see a region name being passed to a boto3 client instantiation anywhere else in the code base, except for direct DSS access and there is a ticket about eliminating that. I you find yourself needing to pass a region name on your machine/personal deployment, you probably didn't set AWS_DEFAULT_REGION as per the example deployment.

Note this used to be set in [environment.local](https://github.com/DataBiosphere/azul/blob/a49b098a38772d10b06594636e323e76e214f2bc/deployments/.example.local/environment.local#L2) but on develop this now is in [environment](https://github.com/DataBiosphere/azul/blob/hca/develop/deployments/.example.local/environment.local#L2).